core/2d/MenuItem.cpp

```diff
@@ -148,7 +149,7 @@ void MenuItemLabel::setLabel(Node* var)
     _label = var;
 }
 
-MenuItemLabel* MenuItemLabel::create(Node* label, const ccMenuCallback& callback)
+MenuItemLabel* MenuItemLabel::create(Node* label, const menuCallback& callback)
```

Comment:
the type shoud starts with uppercase

Reply:
Even though menuCallback is just a typedef alias for `std::function<void(Ref*)>`?

Reply:
maybe use common callback is better

Reply:
maybe: InputCallback

Reply:
There be 3 same typedef
![image](https://github.com/axmolengine/axmol/assets/8652787/7a8ec67f-5707-4383-bc18-4c7ef654d10a)

Reply:
I vote for just getting rid of `cc` prefix, ie leave `MenuCallback`, `WidgetClickCallback`, etc.